parser.y

```diff
@@ -2208,10 +2213,63 @@ BeginTransactionStmt:
 	{
 		$$ = &ast.BeginStmt{}
 	}
+|	"START" "TRANSACTION" "READ" "WRITE"
+	{
+		$$ = &ast.BeginStmt{}
+	}
 |	"START" "TRANSACTION" "WITH" "CONSISTENT" "SNAPSHOT"
 	{
 		$$ = &ast.BeginStmt{}
 	}
+|	"START" "TRANSACTION" "READ" "ONLY"
+	{
+		$$ = &ast.BeginStmt{
```

Comment:
TiDB should handle this later on, because MySQL also has "start transaction read only" and we should keep the compatibility.

Reply:
'START TRANSACTION READ ONLY' is parsed separately which doesn't specify any Bound therefore can be have different execution plan on TiDB.

In addition it would have the same semantic as 'START TRANSACTION READ ONLY WITH TIMESTAMP BOUND STRONG'. Therefore we can add them together in this PR.